_inc/client/plans/accented-card.jsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const AccentedCard = props => {
```

Comment:
That is my plan, but I don't want to move it in there until I decide on a pattern for setting the accent color. I figured I would do that in a followup PR since this one has gotten large.

Reply:
Yes, that works for me!

Reply:
Thanks! I guess keeping this PRopen  as we land some of it in separate PRs won't hurt anyone, right?